crates/agent-sql/src/publications.rs

```diff
@@ -287,7 +307,7 @@ pub async fn find_tenant_quotas(
                     where
                         live_specs.spec_type = 'capture' or
                         live_specs.spec_type = 'materialization' or
-                        live_specs.spec_type = 'collection' and live_specs.spec->'derivation' is not null
```

Comment:
I seem to remember something about how we still technically support `derivation` for backwards compatibility, right? Does that matter here (or have we since actually gotten rid of it?)

Reply:
yeah, we no longer have any specs in prod that use `derivation`.